src/main/python/pybuilder/pip_utils.py

```diff
@@ -41,10 +41,69 @@ def build_dependency_version_string(mixed):
     return version
 
 
-def pip_install(install_targets, index_url=None, extra_index_url=None, upgrade=False, insecure_installs=None,
-                force_reinstall=False, target_dir=None, verbose=False, trusted_host=None, constraint_file=None,
+def pip_install_get_output(install_targets, index_url=None,
```

Comment:
Please put each argument on a single line, that makes it easier to read.

Reply:
Sorry, I've just checked that I had my pyflake set up line shorter than the one specified in pybuilder.